kitsune/products/management/commands/sync_product_versions.py

```diff
@@ -0,0 +1,228 @@
+import re
+
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+from product_details import product_details
+
+from kitsune.products.models import Product, Version
+
+
+class Command(BaseCommand):
+    help = "Sync product versions from Mozilla product-details"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--product",
+            type=str,
+            help="Sync only this product (slug). Options: firefox, mobile, thunderbird",
+        )
+        parser.add_argument(
+            "--dry-run",
+            action="store_true",
+            help="Show what would be changed without modifying the database",
+        )
+
+    def handle(self, *args, **options):
+        dry_run = options["dry_run"]
```

Comment:
Based on the code below and some testing, the `dry-run` option doesn't show what would have changed.